_redirects

```diff
@@ -15,6 +15,9 @@ https://babel.netlify.com/* https://babeljs.io/:splat 301!
 /7/                                 https://old.babeljs.io/repl/build/7.0/
 /7.html                             https://old.babeljs.io/repl/build/7.0/
 
+/docs/en/plugins#pluginpreset-paths /docs/en/plugins#using-a-plugin
+/docs/en/presets#preset-paths /docs/en/presets#using-a-preset
```

Comment:
not sure if this works?

Reply:
I don't think so, the hash is not sent to the server.
We can add an empty div with `id="preset-paths"` next to the `using-a-preset` section.

Reply:
Awesome! 
It's a little below the section, but it's not a big deal if we don't either way I guess?